setup.py

```diff
@@ -29,6 +29,14 @@
     raise RuntimeError("Python >= 3.8 is required")
 
 
+def copy_data_tree(datadir, destdir):
+    try:
```

Comment:
Suggest test for presence rather than via exception:

```Python
if os.path.exists(destdir):
    shutil.rmtree(destdir)
shutil.copytree(datadir, destdir)
```

Reply:
Or just not try to remove it at all? In theory, for building a wheel, it should never already be present?

Reply:
> for building a wheel, it should never already be present?

Actually, pip moved from out-of-tree builds in temporary directory to inplace builds, so yes this can happen if you do multiple builds, and so should remove the directory.